source/extensions/common/wasm/wasm.cc

```diff
@@ -930,7 +936,14 @@ Wasm::Wasm(absl::string_view vm, absl::string_view id) {
   wasm_vm_ = Common::Wasm::createWasmVm(vm);
   id_ = std::string(id);
   if (wasm_vm_) {
+
 #define _REGISTER(_fn) registerCallback(wasm_vm_.get(), #_fn, &_fn##Handler);
+    _REGISTER(getTotalMemory);
+    _REGISTER(_emscripten_get_heap_size);
+#undef _REGISTER
+
+    // Calls with the "_proxy_" prefix.
+#define _REGISTER(_fn) registerCallback(wasm_vm_.get(), "_proxy_" #_fn, &_fn##Handler);
```

Comment:
nit:Could you rename to _REGISTER_PROXY? 
It is not code search friendly as others need extra efforts to differentiate between the "_proxy_" registration and the above EMPTY registration

Reply:
SGTM.  Why don't you issue a PR for the change. :)  I accidentally closed this PR when I thought I was just squashing your changes into mine.